templates/repo/issue/labels/labels_selector_field.tmpl

```diff
@@ -0,0 +1,45 @@
+<div class="ui {{if or (not .HasIssuesOrPullsWritePermission) .Repository.IsArchived}}disabled{{end}} floating jump select-label dropdown">
+	<a class="text gt-df gt-ac muted">
+		<strong>{{.locale.Tr "repo.issues.new.labels"}}</strong>
+		{{if and .HasIssuesOrPullsWritePermission (not .Repository.IsArchived)}}
+			{{svg "octicon-gear" 16 "gt-ml-2"}}
+		{{end}}
+	</a>
+	<div class="filter menu" {{if .Issue}}data-action="update" data-issue-id="{{$.Issue.ID}}" data-update-url="{{$.RepoLink}}/issues/labels"{{else}}data-id="#label_ids"{{end}}>
+		<div class="header" style="text-transform: none;font-size:16px;">{{.locale.Tr "repo.issues.new.add_labels_title"}}</div>
+		{{if or .Labels .OrgLabels}}
+			<div class="ui icon search input">
+				<i class="icon gt-df gt-ac gt-jc">{{svg "octicon-search" 16}}</i>
+				<input type="text" placeholder="{{.locale.Tr "repo.issues.filter_labels"}}">
```

Comment:
By the way, isn't that rather 
```suggestion
				<input type="search" placeholder="{{.locale.Tr "repo.issues.filter_labels"}}">
```
?
I've just noticed that this input type exists, but perhaps we should convert all existing search fields in another PR.

Reply:
> `<input type="search">` elements of type search are `text` fields designed for the user to enter search queries into. These are functionally identical to text inputs

I do not think it's necessary to change the type. And this element is managed by Fomantic UI, so keeping it as Fomantic documented example is better.